lib/dpla/conditions.rb

```diff
@@ -11,13 +11,19 @@ def to_s
     private
 
     def transform(conditions)
+      # When is `conditions' not an array or hash?  It seems that it should
+      # always be one or the other, based on what I see elsewhere in the
+      # code. It can be an array of IDs when called by DPLA::Items.by_ids.
+      # It is usually a Hash.  --Mark B
```

Comment:
I likewise could not find any case in which `conditions` was neither an Array nor a Hash.